### ISSUE TITLE
ERROR > command 'HookyQR.beautifyFile' not found

### ISSUE THREAD

**Issue by vdurante** [HookyQR/VSCodeBeautify#144]
**I have just upgraded to beautify 1.1.0**

* **Operating System (+Version)**:  Windows 10 Build 15063
* **VS Code Version**: 1.12.2
* **beautify Version**: 1.1.0

VS Code:
```javascript
{
    "workbench.iconTheme": "vscode-icons",
    "workbench.colorTheme": "One Dark Pro",
    "window.zoomLevel": 0,
    "html.format.wrapAttributes": "force-aligned",
    "editor.fontFamily": "Consolas, 'Courier New', monospace",
    "editor.tabSize": 2,
    "workbench.sideBar.location": "right"
}
```
.jsbeautifyrc
```javascript
{
  "wrap_attributes": "force-aligned"
}
```

.editorconfig
```
root = true

[*]
charset = utf-8
indent_style = space
indent_size = 2
insert_final_newline = true
trim_trailing_whitespace = true

[*.md]
max_line_length = off
trim_trailing_whitespace = false
```

## Action performed
Format any file with `HookyQR.beautifyFile` command
F1 > Beautify file

## Expected results
Expected formatted document

## Actual results
ERROR > command 'HookyQR.beautifyFile' not found

## Extra dumb question not related to the issue
Whenever I `ALT+SHIFT+F`, the file is not formated using `.jsbeautifyrc` configuration. Is this expected behavior? Since it appears that `ALT+SHIFT+F` command formats the document using VSCodeBeautify.

**Comment by steven11329** [comment]
I have same problem.

**Comment by maeganjwilson** [comment]
I have the same problem also

**Comment by vdurante** [comment]
Apparently I solved it by going to `%USERPROFILE%\.vscode\extensions\HookyQR.beautify-1.1.0` in a command prompt and running `npm install --production`

**Comment by RPaez09** [comment]
Last time this problem occurred, @HookyQR was able to fix it easily. In the mean time, the solution provided by @vdurante above worked for me and It'll work for you too.

Thank you @vdurante

**Comment by stgogm** [comment]
@vdurante The head banging did the trick. Thank you!

**Comment by olegmerinov** [comment]
@vdurante works, as well

**Comment by rob-weiss** [comment]
That fixed it on a Mac as well. I used `cd ~/.vscode/extensions/HookyQR.beautify-1.1.0` to navigate to the Beautify folder, ran `npm install --production` as described above, and restarted Visual Studio Code.

Thanks @vdurante!

**Comment by Favna** [comment]
@vdurante many thanks for the temporary solution. My code was getting so dirty :(

**Comment by HookyQR** [comment]
Fixed in 1.1.1. I really have to track down and remove that old version of VSCE that my laptop keeps finding. 😞  Sorry everyone.
